packages/ui-react/src/components/CustomRegisterField/__snapshots__/CustomRegisterField.test.tsx.snap

```diff
@@ -283,8 +283,6 @@ exports[`<CustomRegisterField> > renders and matches snapshot <TextField> 1`] =
     >
       <input
         aria-describedby="helper_text_1235_name"
-        aria-invalid="false"
```

Comment:
Has `aria-invalid` been removed here entirely?

Reply:
I see that you changed the prop to be exclusively `true`

Reply:
Should not be the case. I will investigate

Reply:
Even though it doesn't matter if `aria-invalid="false" or not present, because it has the same meaning. This was not my intention.
I thought the Boolean casting was redundant, so I removed it. But I have introduced it again in my latest commit to keep it as it previsouly was.